homeassistant/components/nfandroidtv/notify.py

```diff
@@ -266,7 +201,8 @@ def load_file(
             if local_path is not None:
                 # Check whether path is whitelisted in configuration.yaml
                 if self.is_allowed_path(local_path):
-                    return open(local_path, "rb")  # pylint: disable=consider-using-with
+                    with open(local_path, "rb") as path_handle:
```

Comment:
If we use a context manager here the file handle will be closed after returning the file handle. That is probably not what we want.